features/steps/crash_assertion_steps.rb

```diff
@@ -2,16 +2,27 @@
   # Two different outcomes can arise from this scenario, either:
   # * Write will fail on non-writable memory
   # * malloc fails in NSLog
+  #
+  # Depending on OS version, this changes the stacktrace contents
   body = find_request(0)[:body]
   exception = read_key_path(body, "events.0.exceptions.0")
   stacktrace = exception["stacktrace"]
   assert_true(stacktrace.length > 0, "The stacktrace must have more than 0 elements")
 
   case stacktrace.first["method"]
-  when "__pthread_kill"
+  when "__pthread_kill" # Any
     assert_equal(exception["errorClass"], "SIGABRT")
     assert_equal(stacktrace[1]["method"], "abort")
-  when "_nc_table_find_64"
+  when "nanov2_allocate_from_block" # iOS 12.1
```

Comment:
there are seems two cases for iOS 12.1, is it correct?
Can we case switch depending on the simulator runtime instead of first method?
Like `case MAZE_SDK` or `case iOS_version`

Reply:
Yeah, that sounds good 👍